src/main/java/at/hannibal2/skyhanni/test/TestCopyRngMeterValues.kt

```diff
@@ -21,11 +21,11 @@ object TestCopyRngMeterValues {
     private val patternGroup = RepoPattern.group("test.dev.copyrng")
     private val slayerPattern by patternGroup.pattern(
         "slayer",
-        "§7Slayer XP: §d.*§5/§d(?<xp>.*)"
+        "§7§7Slayer XP: §d.*§5/§d(?<xp>.*)"
     )
     private val dungeonPattern by patternGroup.pattern(
         "dungeon",
-        "§7Dungeon Score: §d.*§5/§d(?<xp>.*)"
+        "§7§7Dungeon Score: §d.*§5/§d(?<xp>.*)"
```

Comment:
Please improve your code as dictated below:
```suggestion
        "(§7){2}Dungeon Score: §d.*§5/§d(?<xp>.*)"
```

Reply:
i will not do this

Reply:
![grafik](https://github.com/user-attachments/assets/b50fb710-0219-4aad-8df2-04e8a748286b)

Reply:
Please move all dungeon related repo patterns in the new utils file RepoPatternsDungeon.kt